src/vs/code/electron-main/window.ts

```diff
@@ -84,6 +84,7 @@ export interface IWindowConfiguration extends ParsedArgs {
 	perfWindowLoadTime?: number;
 
 	workspacePath?: string;
+	backupFolder?: string;
```

Comment:
I removed this specific member in https://github.com/Microsoft/vscode/pull/16303/commits/10c12111a54d8d35aeedabe7a5a2e1a85ac12559

On the naming, folder is used in some placed because it's only folder, as opposed to the full path.